application_service/app_service.mak

```diff
@@ -33,7 +33,6 @@ I= $(SRC_DIR)/include
 INCLUDE= -I$(I) -I$(LIBSRC)/sev-snp -I/usr/local/opt/openssl@1.1/include/
 
 CFLAGS=$(INCLUDE) -O3 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -D DEBUG -Wno-deprecated-declarations
-CFLAGS1=$(INCLUDE) -O1 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -D DEBUG -Wno-deprecated-declarations
```

Comment:
You will see some lines like this one being deleted. I found that definitions such as `CFLAGS1` are unused, so took the opportunity to cleanup the Makefile by deleting such unused definitions and unused targets (one case).